samples/snippets/snippets_mute_config.py

```diff
@@ -0,0 +1,236 @@
+
+    mute_config = client.get_mute_config(request)
+    print(f"Retrieved the mute config: {mute_config}")
+
+
+# [END securitycenter_get_mute_config]
+
+
+# [START securitycenter_list_mute_configs]
+def list_mute_rules(parent: str) -> None:
+    """
+    Listing mute configs at organization level will give all the configs
+    at org, folder and project levels.
+    Similarly, listing configs at folder level will list all the configs
+    at folder + projects (within that folder) level.
+    Args:
+        parent: Use any one of the following three resource paths to list mute configurations.
+        "organizations/{organization_id}" or
+        "folders/{folder_id}" or
+        "projects/{project_id}"
+    """
+    from google.cloud import securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+
+    request = securitycenter.ListMuteConfigsRequest()
+    request.parent = parent
+
+    # List all Mute Configs present in the resource.
+    for mute_config in client.list_mute_configs(request):
+        print(mute_config.name)
+
+
+# [END securitycenter_list_mute_configs]
+
+
+# [START securitycenter_update_mute_config]
+def update_mute_rule(mute_config_name: str) -> None:
+    """
+    Updates an existing mute configuration.
+    The following can be updated in a mute config: description, and filter/ mute rule.
+    Args:
+        mute_config_name: Specify the name of the mute config to delete.
+        Its format is:
+        organizations/{organization}/muteConfigs/{config_id} or
+        folders/{folder}/muteConfigs/{config_id} or
+        projects/{project}/muteConfigs/{config_id}
+    """
+    from google.cloud import securitycenter
+    from google.protobuf import field_mask_pb2
+
+    client = securitycenter.SecurityCenterClient()
+
+    update_mute_config = securitycenter.MuteConfig()
+    update_mute_config.name = mute_config_name
+    update_mute_config.description = "Updated mute config description"
+
+    field_mask = field_mask_pb2.FieldMask(paths=["description"])
+
+    request = securitycenter.UpdateMuteConfigRequest()
+    request.mute_config = update_mute_config
+    # Set the update mask to specify which properties of the Mute Config should be updated.
+    # If empty all mutable fields will be updated.
+    # Make sure that the mask fields match the properties changed in 'update_mute_config'.
+    # For more info on constructing update mask path, see the proto or:
+    # https://cloud.google.com/security-command-center/docs/reference/rest/v1/folders.muteConfigs/patch?hl=en#query-parameters
+    request.update_mask = field_mask
+
+    mute_config = client.update_mute_config(request)
+    print(f"Updated mute config : {mute_config}")
+
+
+# [END securitycenter_update_mute_config]
+
+
+# [START securitycenter_set_mute_unmute]
+def set_mute_unmute_finding(finding_path: str) -> None:
+    """
+      Mute/unmute an individual finding.
+      If a finding is already muted, muting it again (or vice versa) is a no-op.
```

Comment:
If a finding is already muted, muting it again has no effect. Similarly, unmuting a finding that isn't muted has no effect.